bitnami/solr/values.yaml

```diff
@@ -0,0 +1,750 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
```

Comment:
I still don't see, for example, cloudBootstrap here. The results of my script again. Ignore the false positives

```
Element cloudBootsrap in README not found in values.yaml
Element existingSecret in README not found in values.yaml
Element exporter.image.pullSecrets in README not found in values.yaml
Element exporter.service.loadBalancerIP in README not found in values.yaml
Element fullnameOverride in README not found in values.yaml
Element global.imagePullSecrets in README not found in values.yaml
Element global.imageRegistry in README not found in values.yaml
Element global.storageClass in README not found in values.yaml
Element ingress.extraHosts[0].name in README not found in values.yaml
Element ingress.extraHosts[0].path in README not found in values.yaml
Element ingress.extraPaths in README not found in values.yaml
Element ingress.extraTls[0].hosts[0] in README not found in values.yaml
Element ingress.extraTls[0].secretName in README not found in values.yaml
Element ingress.secrets[0].certificate in README not found in values.yaml
Element ingress.secrets[0].key in README not found in values.yaml
Element ingress.secrets[0].name in README not found in values.yaml
Element nameOverride in README not found in values.yaml
Element persistence.matchLabels in README not found in values.yaml
Element persistence.storageClass in README not found in values.yaml
Element podSecurityPolicy.create in README not found in values.yaml
Element service.loadBalancerIP in README not found in values.yaml
```